Cargo.toml

```diff
@@ -26,7 +26,13 @@ anyhow = "1.0"
 arbitrary = { version = "1", features = ["derive"] }
 assert2 = "0.3"
 colored = "2.1"
-clap = { version = "4", features = ["derive", "cargo", "wrap_help", "unicode", "string"] }
+clap = { version = "4", features = [
+    "derive",
+    "cargo",
+    "wrap_help",
+    "unicode",
+    "string",
+] }
```

Comment:
Why the reformat?

Reply:
My IDE did that. I don't have a pronounced opinion.